### DISCUSSION TITLE
河神 | /cn/2022/07/river-god/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1448] [Comments]
https://yihui.org/cn/2022/07/river-god/

[a comment by an automated account is not shown]

**Comment by elizen** [comment]
是不是应该更新了？每年一次期待，感觉今年晚了一点点。

**Reply by yihui** [reply]
嗯，这一年没写多少，而且这两个月在休假中，我还没决定今年是否更新。

**Reply by ZaiyanHere** [reply]
更新一下吧！我来促进一下你的决定：）

**Reply by yihui** [reply]
促得很好，下次不要促了。

我已决定明年再更。今年阅读时间不够，太多文章和笔记都还没整理。请各位客官各回各家、各找各妈、该干啥干啥。

**Reply by ZaiyanHere** [reply]
那祝愿yihui接下来的日子里时间充足 悠闲自在：）

**Reply by yihui** [reply]
跨越重重艰难险阻，风一更，雪一更，终于已更！

> 山一程，水一程，身向榆关那畔行，夜深千帐灯。  
> 风一更，雪一更，聒碎乡心梦不成，故园无此声。

**Comment by gaospecial** [comment]
你咋好几年都不带更新了？

**Reply by yihui** [reply]
才三年而已嘛。放心，终有一天会更新的。

**Reply by liechi** [reply]
是三年不更新，一更看三年那种吗？

**Reply by yihui** [reply]
看不了三年。三天应该足以看完，但少数文章可能会回味三年。初步计划本周更新。过去两年几乎没写，直到上个月才重新接续；现在要写的已经差不多写完了，新写的东西会是一个全新的文体，一个月之前我自己都不知道我会这样写。总之，请准备好去取钱回来捧个钱场。

**Reply by gaospecial** [reply]
你看，一旦没了更新 KPI，人就会变得懒惰。

**Reply by yihui** [reply]
写而不 K 则爽，K 而不写则殆。
